build.gradle

```diff
@@ -114,8 +137,7 @@ jar {
     }
 
     // Copy libs jar files
-    from { configurations.external.collect { it.isDirectory() ? it : zipTree(it) } }
-    exclude(['dan200/**', 'ic2/**', 'li/**', 'buildcraft/**'])
+    // from { configurations.external.collect { it.isDirectory() ? it : zipTree(it) } }
```

Comment:
Nit: Can probably just delete this, right?

Reply:
As of right now yes, but didn't know if you want to shade some other stuff in later or something. If you want to shade much more you really should make a shade configuration (again see GT6's build script, or upgrade to RFG's build system).